packages/cicero-core/src/instance.js

```diff
@@ -161,8 +182,21 @@ class TemplateInstance {
         this.parserManager.setCurrentTime(currentTime, utcOffset);
 
         // Draft
-        const ciceroMark = this.templateMarkTransformer.draftCiceroMark(data, this.parserManager, templateKind, {});
-        return this.formatCiceroMark(ciceroMark,options);
+        const ciceroMark = this.templateMarkTransformer.draftCiceroMark(data, this.parserManager, kind, {});
+        const draft =  this.formatCiceroMark(ciceroMark,options);
+
+        //Add state
+        if (kind === 'contract') {
```

Comment:
I really think we need a spec or some docs around what we are putting in the history file, the format of that file and who is the intended consumer (human vs machine).

Reply:
Agreed. Let's merge this PR into the `poc-slc` and then iterate on the final spec.